Integrations/integration-ArcherRSA.yml

```diff
@@ -2387,46 +2387,85 @@ script:
         })
     }
 
-    function getGroupIDFromGroupName(groupsDataFromArcher, desiredGroupName) {
+    function getGroupIDFromGroupName(groupsDataFromArcher, userSuppliedGroupName) {
         var desiredGroupID = -1;
         for (var i=0; i<groupsDataFromArcher.length; i++){
-            var currentGroupData = groupsDataFromArcher[i].RequestedObject;
-            var currentGroupName = currentGroupData.Name;
-            if (currentGroupName.trim() === desiredGroupName.trim()){
-                desiredGroupID = currentGroupData.Id;
+            var currentArcherGroupData = groupsDataFromArcher[i].RequestedObject;
+            var currentArcherGroupName = currentArcherGroupData.Name;
+            if (currentArcherGroupName.trim() === userSuppliedGroupName.trim()){
+                desiredGroupID = currentArcherGroupData.Id;
             }
         }
         return desiredGroupID;
     }
 
+
+    function getUserIDFromUserName(allUsersArcherData, userSuppliedUserName) {
```

Comment:
same comments

Reply:
same to what?

Reply:
same to `getGroupIDFromGroupName` which is the implements the exactly same functionality

Reply:
Fixed